package.json

```
@@ -246,7 +234,7 @@
             "title": "Configuring Waterproof Settings (Path)",
             "when": "isMac && !hasRemoteServer",
             "id": "configure-settings-mac",
-            "description": "Change the [Waterproof.Path](command:waterproof.pathSetting) setting.\n[Use Default](command:waterproof.defaultPath)\n[Change Setting](command:waterproof.pathSetting)",
+            "description": "If necessary (i.e. if running `which coq-lsp` does not turn up anything when run from a terminal within vscode) change the path to the `coq-lsp` program (you can find this path by running `which coq-lsp` in the terminal in which you installed `coq-lsp`).\n[Change Setting](command:waterproof.pathSetting)",
```

```suggestion
            "description": "If necessary (i.e. Waterproof does not start) change the path to the `coq-lsp` program (you can find this path by running `which coq-lsp` in the terminal in which you installed `coq-lsp`).\n[Change Setting](command:waterproof.pathSetting)",
```

The problem here is that it's often the case that the terminal in VSCode will have the opam switch correctly configured, but the vs code instances as a whole doesn't.